week05/store_test.go

```diff
@@ -0,0 +1,176 @@
+package demo
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestStoreDb(t *testing.T) {
+	t.Parallel()
+
+	tcases := []struct {
+		name  string
+		store Store
+		want  data
+	}{
+		{"empty", Store{}, data{}},
+		{"empty", Store{data: data{"cars": Models{{"A": "B"}}}}, data{"cars": Models{{"A": "B"}}}},
+	}
+
+	for _, tt := range tcases {
+		t.Run(tt.name, func(t *testing.T) {
+			got := tt.store.db()
+			if !reflect.DeepEqual(got, tt.want) {
```

Comment:
You should not be using `reflect`. There is no need to use `reflect` to complete this assignment.